lib/get-file-ending.js

```diff
@@ -0,0 +1,31 @@
+var fs = require('fs');
+
+
+module.exports = function getFileEnding(candidate, cb) {
```

Comment:
I would suggest renaming `getFileEnding` to something that better reflects what it does. This function is clearly specific to markdown files, and does more than finding the "ending" for file names: it checks that these files actually exists. 

Maybe `findMarkdownFile` with a cb that takes and error and the file name (including the extension) of a markdown file that exists would be clearer?